### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by rahulcs754**
- link correction
- word correction
image

### CODE REVIEW

File: docs/intro/pages/Navigation.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h1 class="header-logo"><a href="#">RS UI </a></h1>
 
         <ul class="header-nav">
             <li><a href="https://rs-ui.netlify.app/"> Home </a></li>
-            <li><a href="https://rs-ui.netlify.app/textutility.html"> Component List </a></li>
+            <li><a href="https://rs-ui.netlify.app/docs/pages/textutility.html"> Component List </a></li>
         </ul>
```
Comment (shubhamkumar648) [inline docs/intro/pages/Navigation.html:33]: avoid space

**Review by rittik-ghosh — COMMENTED** [review]
Only Links are changed along with few word correction.
All the files are evenly oriented and look good

**Review by shubhamkumar648 — APPROVED** [review]
code looks clean to me. just avoid space,

File: docs/intro/pages/Navigation.html (hunk 1 above)
Comment (rahulcs754) [reply docs/intro/pages/Navigation.html:33]: done

**Pull request merged**